### PR TITLE
Arp scan specific interface

### PR DESCRIPTION
**Pull request by NiklasNeugebauer**
Adds ability to specify a specific network interface to use when scanning for cameras.

Branches from #109

### CODE REVIEW

File: rosys/vision/rtsp_camera/arp_scan.py
Comment (pascalzauberzeug) [inline rosys/vision/rtsp_camera/arp_scan.py:1]: We should decide whether to use the term 'interface' or 'adapter'.
Arp-scan and the old library netifaces use 'interface' and ifaddr uses 'adapter'.
Personally, I would go with 'interface'

Comment (rodja) [reply rosys/vision/rtsp_camera/arp_scan.py:1]: I also prefere 'interface'. Its more in line with unix nomeclatur.

Comment (pascalzauberzeug) [reply rosys/vision/rtsp_camera/arp_scan.py:1]: I renamed it

**Review by pascalzauberzeug — APPROVED** [review]
I resolved the conflicts with main, looks good to me

**Pull request merged**